cdap-data-fabric/src/main/java/co/cask/cdap/data2/admin/DefaultAdmin.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.internal.app.runtime;
+package co.cask.cdap.data2.admin;
```

Comment:
Why move the data2? The Admin is not just for data-fabric. It is intended for any future admin functions addition.